site/confirmdelete.html.jet

```diff
@@ -1,6 +1,7 @@
 {{extends "templates/application/application.jet"}}
 
 {{block head()}}
+  <script>localStorage.removeItem('shift72.authToken')</script>
```

Comment:
Am I putting in another hack here to get rid of the nav bar, well I assume it would be a hack. Or is making sure they're logged out good enough.

Reply:
I think for now just making sure they are signed out is enough. But Sam was the one pushing for the no title bar so will leave it for him to comment.

Reply:
Removing the header / footer / nav blocks was an easy solution although there was a slight weirdness with the mobile CSS when you did that -- it got rid of the padding at the top of the page and it looked weird with the title squished right at the top but no nav.

I think the current solution is good enough, especially if they're logged out.

Maybe if we do an account/auth micro-app (same approach as checkout) then we can move this functionality in there.